### DISCUSSION TITLE
Can not connect to remote Database. Getting the next error: failed to create migration table: ERROR: permission denied for schema supabase_migrations (SQLSTATE 42501)

### DISCUSSION THREAD

**Discussion by nik-lylo** [supabase/supabase#27567] [Questions]
I am trying to set the local development and connect to the remote database using 
the `supabase link` command. After entering the password I get the next error:

**failed to create migration table: ERROR: permission denied for schema supabase_migrations (SQLSTATE 42501)** .

It is very strange as the superbase executes this operation with USER postgres. And for some reason, USER postgres does not have access to the supabase_migrations schema. Also, I can not grant permission for USER postgres as only superuser can grant permission to schemas that Supabase creates.

By the way, I can not reset the project as I have active users. When I tried to create a new project and link to Supabase everything worked as expected and I connected without any problems.

**Comment by dthib** [comment]
Same issue here. It's for a projet I have just joined so I will try to get more info.
Thanks
